packages/paste-core/components/tooltip/src/Tooltip.tsx

```diff
@@ -82,15 +115,62 @@ const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(
           {/* import Paste Theme Based Styles due to portal positioning. */}
           <StyledBase>
             <TooltipArrow {...tooltip} />
-            <Text
-              element={`${element}_TEXT`}
-              as="span"
-              color="colorTextInverse"
-              fontSize="fontSize20"
-              lineHeight="lineHeight10"
-            >
-              {text}
-            </Text>
+            {text && !keyCombinationsActions && (
+              <Text
+                element={`${element}_TEXT`}
+                as="span"
+                color="colorTextInverse"
+                fontSize="fontSize20"
+                lineHeight="lineHeight10"
+              >
+                {text}
+              </Text>
+            )}
+            {keyCombinationsActions && (
```

Comment:
```suggestion
            {keyCombinationsActions && !text && (
```
question: could this help non-ts users?